src/extension/background-script/connectors/alby.ts

```diff
@@ -64,6 +65,25 @@ export default class Alby implements Connector {
     return this.config.oAuthToken;
   }
 
+  async persistUserIdentifier(userIdentifier: string): Promise<void> {
+    if (!this.config.userIdentifer) {
+      if (this.account.id) {
+        const accounts = state.getState().accounts;
+        const password = (await state.getState().password()) as string;
+
+        const configData = decryptData(
+          accounts[this.account.id].config,
+          password
+        );
+        configData.userIdentifer = userIdentifier;
+        accounts[this.account.id].config = encryptData(configData, password);
+        state.setState({ accounts });
+        // make sure we immediately persist the updated accounts
+        await state.getState().saveToStorage();
+      }
```

Comment:
we repeat this a few times it seems, can we put this into it's own function?

Reply:
here we 
1. decrypt
2. add or delete something in account config
3 encrypt it again. 

2nd step can always be anything that is why I didn't refactored it much. also currently we need userIdentifier only for alby connector so didn't make it universal. 

can we avoid doing any change here?